pkg/op-app/op_app/api.py

```diff
@@ -73,12 +74,22 @@ def handle_exception(e):
     return jsonify({"error": "An unexpected error occurred during prediction"}), 500
 
 
-@app.route("/predict", methods=["POST"])
-@rate_limit(100, timedelta(minutes=1))
-@handle_question
-async def predict(question, memory):
-    result = await process_question(question, memory)
+async def capture_question_classification(question, memory, user_token):
+    result = await classify_question(question, memory)
+    classification = result["classification"]
+
+    posthog.capture(
+        user_token,
+        "QUESTION_CLASSIFICATION",
```

Comment:
Why emit a new event? Couldn't we just put this in the event that already exists?

Reply:
we can do it 👍